### PR TITLE
Restructured Project

### PR DESCRIPTION
**Pull request by clintonb**
Moved the Django apps to the root level. This is actually the norm for Django projects. Also, this will enable us to share code common code amongst the apps and the acceptance tests.

@dsjen @jbau @dylanrhodes

### CODE REVIEW

**Comment by clintonb** [comment]
Any feedback on this? It needs to happen sooner--before graded content is merged--rather than later.

**Comment by e0d** [comment]
@clintonb I assume the deployment scripts will need some updates.  True?

**Comment by clintonb** [comment]
@e0d Yes. Thanks for the reminder.

**Comment by dsjen** [comment]
Thanks for the reminder--taking a look today. :smile:

**Comment by dsjen** [comment]
Got wrapped up with something else.  I'll check it first thing tomorrow.

File: acceptance_tests/pages.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ def __init__(self, browser, course_id=None):
         self.page_url += '/enrollment/activity/'
 
     def is_browser_on_page(self):
-        return super(CourseEnrollmentActivityPage, self).is_browser_on_page() and \
-               'Enrollment Activity' in self.browser.title
+        return super(CourseEnrollmentActivityPage,
+                     self).is_browser_on_page() and 'Enrollment Activity' in self.browser.title
```
Comment (dsjen) [inline acceptance_tests/pages.py]: Out of curiosity, are these changes in response to linting errors?

Comment (clintonb) [inline acceptance_tests/pages.py]: Yes.

**Comment by dsjen** [comment]
The previous structure followed recommendations from Two Scoops of Django, and their reasoning was that it makes it makes it clearer which are app directories and resources.  They did note that there are many other structures.

The changes look good to me.  Thanks for tackling it!

**Pull request merged**